docs/external-disk.md

```diff
@@ -19,45 +19,112 @@ The `-disk` specification may be repeated for multiple disks, although a limited
 
 **TODO:** GCP
 
-## Mount the Disk
-A disk created or used via `hyperkit run` will be available inside the image at `/dev/vda` with the first partition at `/dev/vda1`.
+## Format the disk
 
-In order to use the disk, you need to do several steps to make it available:
+`pkg/format` creates a partition table and format drives for use with LinuxKit
 
-1. Create a partition table if it does not have one.
-2. Create a filesystem if it does not have one.
-3. `fsck` the filesystem.
-4. Mount it.
+### Example Usage
 
-To simplify the process, two `onboot` images are available for you to use:
+This packages supports two modes of use:
 
-1. `format`, which:
-    * checks for a partition table and creates one if necessary
-    * checks for a filesystem on the partition and creates one if necessary
-    * runs `fsck` on the filesystem
-2. `mount` which mounts the filesystem to a provided path
+```
+onboot:
+  name: format
```

Comment:
the syntax here and for all the examples is wrong: it's missing the `-` in front of name